src/app/core/store/users/users.state.ts

```diff
@@ -22,31 +22,50 @@ import {createSelector} from '@ngrx/store';
 import {AppState} from '../app.state';
 import {UserModel} from './user.model';
 import {selectOrganizationByWorkspace} from "../organizations/organizations.state";
-import {selectAllGroups} from "../groups/groups.state";
-import {UserFilters} from "./user.filters";
+import {selectAllGroups, selectGroupsDictionary} from "../groups/groups.state";
+import {filterUserFunctions, filterUsersByOrganization} from "./user.filters";
 import {GroupModel} from "../groups/group.model";
+import {OrganizationModel} from '../organizations/organization.model';
+import {Dictionary} from '@ngrx/entity/src/models';
+import {isNullOrUndefined} from 'util';
 
 export interface UsersState extends EntityState<UserModel> {
-
+  loadedForOrganizationId: string;
```

Comment:
I am not sure if this is the right approach. Which organization are the users loaded from?

Reply:
As I mention before, user groups is in strange format. It is loaded only with one organization and corresponding groups ids. So I need to know for which organizations are users loaded.

Reply:
Oh, I see. I just didn't get it from the name of the property.